tests/core/test_model_manager.py

```diff
@@ -185,6 +194,7 @@ def test_import_module_registry(self):
 TEST_DATA_PATH = os.path.join("tests", "fixtures")
 DUMMY_LOCAL_MODEL_NAME = "dummy_module_foo"
 DUMMY_BACKEND_MODEL_NAME = "dummy_module_backend"
+DUMMY_NO_ID_MODEL_NAME = "dummy_module_no_id"
```

Comment:
nit: this doesn't seem used - was this intended for the added test?

Reply:
Nice catch, thank you, I defined it and forgot to use it in the test 🤦 
Fixed now